scripts/git-tag-parser.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+VERNUM='0|[1-9][0-9]*'
+
+# An example is "set-namespace/v1.2.3"
+GIT_TAG_SEMVER_REGEX="\
+^([a-zA-Z]*(-[a-zA-Z]+)*)\/\
```

Comment:
You may need to accept every valid docker image (`_`, `.` are also valid in image name), or make it less strict.